estuary/utils/story.py

```diff
@@ -0,0 +1,468 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+import abc
```

Comment:
There should be a new line between these since `abc` is part of the standard library and `neomodel` is a third-party package.